catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisDatabaseOperations.java

```diff
@@ -4,28 +4,118 @@
  */
 package com.datastrato.gravitino.catalog.doris.operation;
 
+import com.datastrato.gravitino.catalog.doris.utils.DorisUtils;
 import com.datastrato.gravitino.catalog.jdbc.JdbcSchema;
 import com.datastrato.gravitino.catalog.jdbc.operation.JdbcDatabaseOperations;
 import com.datastrato.gravitino.exceptions.NoSuchSchemaException;
+import com.datastrato.gravitino.exceptions.NoSuchTableException;
+import com.datastrato.gravitino.meta.AuditInfo;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
 
 /** Database operations for Doris. */
 public class DorisDatabaseOperations extends JdbcDatabaseOperations {
-  // TODO: add implementation for doris catalog
+  public static final String COMMENT_KEY = "comment";
 
   @Override
-  public JdbcSchema load(String databaseName) throws NoSuchSchemaException {
-    throw new UnsupportedOperationException();
+  public String generateCreateDatabaseSql(
+      String databaseName, String comment, Map<String, String> properties) {
+    StringBuilder sqlBuilder = new StringBuilder();
+
+    // Append database name
+    sqlBuilder.append(String.format("CREATE DATABASE `%s`", databaseName));
+
+    // Doris does not support setting schema comment, put comment in properties
+    Map<String, String> newProperties = new HashMap<>(properties);
+    if (StringUtils.isNotEmpty(comment)) {
```

Comment:
Will the comment be empty?

Reply:
Thanks for remind. this line has been removed.